sdk/search/azure-search-documents/azure/search/documents/_internal/_search_client.py

```diff
@@ -138,6 +142,68 @@ def search(self, search_text, **kwargs):
 
         :param str search_text: A full-text search query expression; Use "*" or omit this parameter to
         match all documents.
+        :keyword include_total_result_count: A value that specifies whether to fetch the total count of
+        results. Default is false. Setting this value to true may have a performance impact. Note that
+        the count returned is an approximation.
+        :type include_total_result_count: bool
```

Comment:
"keyword" types have to be called "paramtype" (it's a sphinx limitation)
https://github.com/sphinx-doc/sphinx/issues/6763